### ISSUE TITLE
Please Make NVDA work with OEClassic

### ISSUE THREAD

**Issue by Dennisl123** [nvaccess/nvda#6809]
Hi, Please make OE classic accessible with NVDA.

**Comment by ehollig** [comment]
Could you please provide more information? What exactly in this program does not work?

**Comment by Dennisl123** [comment]
One when arrowing throw the message list NVDA doesn’t read what message you’re on.  The options don’t read.  I suspect the import  export and message rules creations don’t read as well.

**Comment by Brian1Gaff** [comment]
I'm sure I saw a thread about this on the users list saying the latest 
version is better but not working well enough yet. This leads me to think 
that much of the problems with this software are the actual way its been 
written, and as we all know retrofitting accessibility to an already coded 
program can be a nightmare. Maybe we could suggest that somebody 
knowledgeable about  apis and what screenreaders need, try it an talk to the 
developers. If they have at least made an attempt they might be glad of some 
help.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
---

**Comment by Adriani90** [comment]
Closing as duplicate of #8195
